### ISSUE TITLE
Incorrect alignment of markers if series has gaps

### ISSUE THREAD

**Issue by shivamsihare** [tradingview/lightweight-charts#464]
**Lightweight Charts Version:**  2.0.0 

**Steps/code to reproduce:**

```
var chart = LightweightCharts.createChart(document.body, {
	width: 600,
  height: 300,
	layout: {
		backgroundColor: '#ffffff',
		textColor: 'rgba(33, 56, 77, 1)',
	},
	grid: {
		vertLines: {
			color: 'rgba(197, 203, 206, 0.7)',
		},
		horzLines: {
			color: 'rgba(197, 203, 206, 0.7)',
		},
	},
	timeScale: {
		timeVisible: true,
    secondsVisible: false,
	},
});

var lineSeries = chart.addLineSeries();

lineSeries.setData([
	{ time: 1556877600, value: 230.12 },
	{ time: 1556881200, value: 230.24 },
	{ time: 1556884800, value: 230.63 },
	{ time: 1556888400, value: 231.35 },
	{ time: 1556892000, value: 232.24 },
	{ time: 1556895600, value: 232.52 }
]);

var lineSeries1 = chart.addLineSeries({
color: "red"
});

lineSeries1.setData([
	{ time: 1556877600, value: 130.12 },
	{ time: 1556881000, value: 130.24 },
	{ time: 1556884800, value: 130.63 },
	{ time: 1556888400, value: 131.35 },
	{ time: 1556892000, value: 132.24 },
	{ time: 1556895600, value: 132.52 }
]);

lineSeries.setMarkers([{
color: "green",
position: "inBar",
shape: "arrowDown",
time: 1556880900
}]);
chart.timeScale().fitContent();

```

**Actual behavior:**

Marker is appearing at the top of chart

**Expected behavior:**

Marker should appear in the lineSeries "inBar"

**Screenshots**

![markerIssue](https://user-images.githubusercontent.com/24588948/83875218-55fda880-a754-11ea-8286-27795c1849d5.PNG)

**CodeSandbox/JSFiddle/etc link:**

https://jsfiddle.net/sihare123/g2t0hfL6/12/

**Comment by timocov** [comment]
The issue here is that series `lineSeries` doesn't have point with time `1556880900`. If you set "correct" time for the series, it'll display as expected.

**Comment by shivamsihare** [comment]
Actually this issue occurs because the other line series has nearer point than the one I am setting marker on. Even if actual point is not there in line series markers work. You can check by removing lineSeries1 in above code. The marker will display appropriately

**Comment by timocov** [comment]
Yes, that's true. It'll be fixed in #466.
